### DISCUSSION TITLE
Stick command for magnetometer calibration not working

### DISCUSSION THREAD

**Discussion by julianschweizer** [iNavFlight/inav#6745] [Q&A] [answered]
I cant get my quad to "magnetometer calibration" via stick commands. No "beeps" after doing stick commands which indicate the start of calibration mode. 

OSD stick command are working. 

Latest iNAV 2.6.1, Mamba F405 MK2 FC (FURYF4OSD). Taranis X9D R9M + R9MX.

Any ideas how to debug/find out whats wrong?

**Comment by JulioCesarMatias** [comment, marked as answer]
hi friend, have you checked if the yaw, pitch and roll pulses are correct? To call the mag calibration, the combination is throttle_high + yaw_high + pitch_low + roll_center. I believe the problem is in your configuration, and not in the fw of iNAV
